homeassistant/components/unifi/config_flow.py

```diff
@@ -164,6 +203,15 @@ async def async_step_site(self, user_input=None):
             errors=errors,
         )
 
+    def _host_already_configured(self, host):
+        """See if we already have a unifi entry matching the host."""
+        for entry in self._async_current_entries():
+            if CONF_CONTROLLER not in entry.data:
```

Comment:
Im planning on flattening the data structure so please look for the domain instead :)

Reply:
Will change it to `if not entry.data` as it was there to make sure we don't lookup keys that don't exist for ignored flows

Reply:
Shouldnt we make sure its the correct domain? This could fail if it looks at anything other than unifi?

Reply:
```
    @callback
    def _async_current_entries(self) -> List[ConfigEntry]:
        """Return current entries."""
        assert self.hass is not None
        return self.hass.config_entries.async_entries(self.handler)
```

```
homeassistant/config_entries.py:    def async_entries(self, domain: Optional[str] = None) -> List[ConfigEntry]:
```

`_async_current_entries` should only return entries for the `unifi` domain if I'm ready in it right?

Reply:
You're right!